src/components/Modal/Modal.mdx

```diff
@@ -223,6 +226,63 @@ export const ModalStory = (args) => {
 
 <ArgTypes of={ModalStories.useModalStory} />
 
+## Confirmation Variant
+
+The "confirmation" `Modal` variant is used when the modal can confirm or revoke the modal's content. Accordingly, there are two buttons
+in the footer instead of the one in the default modal. Each button has its own custom function (`onCancel`, `onConfirm`) that will close the modal.
+The 'X' Close button in the upper right will call the `onCancel` function.
```

Comment:
As well as the ESC button and clicking anywhere outside the modal.  Maybe change this to the following:
```
Additionally, the `onCancel` function will be called when using 
any of the other available methods for closing the modal, 
including clicking the "x" close button in the upper right corner 
of the modal, clicking anywhere outside the modal UI, or using 
the keyboard ESC button.
```